src/foam/u2/borders/TopBorderCard.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'TopBorderCard',
+  extends: 'foam.u2.View',
+
+  css: `
+    ^ {
+      width: 100%;
+      background: #ffffff;
```

Comment:
Will it never need a background, or will the background be different for some cases? I'm thinking a background property would be helpful

Reply:
Couldn't we wrap it in another BackgroundCard border that we add in this PR? Or we could merge the two

Reply:
@sarthak-marwaha the BackgroundCard border has rounded corners so that would need to be configurable on background card. Also the journals will be more complex as a consequence (i.e. needing to write the spec for two nested borders because you want a background color). It also brings to light that padding could be a border, and the question of where to draw that line.

Reply:
True, wrapping would be complicated in jrls, guess we can add a background prop to this. Just for reference wrapping would look like 
`
border: {
  class: 'foam.u2.borders.TopBorderCard',
  color: '#406dea',
  children: [
     { class: 'foam.u2.borders.BackgroundCard', background: '#406dea' }
  ]
},
`
It's not the worst thing but might be better to just add a prop in this case